### PR TITLE
Tracing socket api call

### PR DESCRIPTION
**Pull request by jjr840430**

### CODE REVIEW

**Comment by jjr840430** [comment]
@efiop Can you review why checking has failed?
The error message is as followings:
WARNING: please, no spaces at the start of a line

#82: FILE: test/cmockatest.c:729:
+      cmocka_unit_test(test_rtr_socket),$

File: sock.c [hunk 1]
```diff
@@ -25,130 +25,71 @@
...
 
-	/* Only implemented for IPv4 right now */
-	if (get_tracing_enabled() && address->sa_family == AF_INET) {
-		FILE *config = NULL;
-
-		while (1) {
-			int ret;
-			struct sockaddr match_addr;
-
-			ret = rtr_get_config_multiple(&config, "connect",
```
Comment (riataman) [inline sock.c]: Any reason you removed the connect() redirection? Was working pretty well.

Comment (jjr840430) [inline sock.c]: @riataman I'm going to add it after added socket API functions such as send, recv, sendfrom, recvfrom, setsocketopt, ioctl...

**Comment by jjr840430** [comment]
@efiop Can you check the result of checkpatch?

**Comment by efiop** [comment]
Oh, need to fix coding style in other test, other than cmockatests.c . Let me fix those, I'll push it in a few minutes so you can rebase. Sorry for inconvenience.

**Comment by efiop** [comment]
Fixed coding style in https://github.com/riboseinc/retrace/pull/117 and merged it. Please try to rebase now.

**Comment by jjr840430** [comment]
Hello, Ruslan, Thank you very much.
I'm now at home. I will rebase tomorrow and let you know.

Thanks again.
Jin.

**Pull request closed without merging**